### PR TITLE
Second version of the main page design.

### PR DESCRIPTION
**Pull request by StephaneGrabsia**
This PR is an update of the design of the main page. 

Still need to be improved, not operationnal. 

To test it manually you can launch it locally and check localhost:8080

### CODE REVIEW

**Review by SullivanCastro — DISMISSED** [review]
Good job fellow !

**Review by sachabinder — DISMISSED** [review]
It is globally very good !
I just don't like colors. I suggest you also to add a margin or a padding between the IDE and the navbar.
Otherwise, very good job !

I let you solve conflicts and merge !!

File: front/src/app.js [hunk 1]
```diff
@@ -37,7 +37,43 @@ const theme = createTheme({
...
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
+    'cupidatat non proident, sunt in culpa qui officia deserunt mollit anim'+
+    'id est laborum.' +
+    ' \n \n '+
+    ' Lorem ipsum dolor sit amet, '+
+    'consectetur adipiscing elit, sed do eiusmod tempor incididunt'+
+    ' ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis'+
+    ' nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo'+
+    ' consequat. Duis aute irure dolor in reprehenderit in voluptate velit'+
+    'esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat'+
```
Comment (sachabinder) [inline front/src/app.js:73]: I know that it is for dev, but it has nothing to do here !!
The `app.js` file is not made for that. Ok for know, but please remove it very quickly.

**Review by sachabinder — APPROVED** [review]
Please check your console to fix problems in the future before PR otherwise, LGTM

**Pull request merged**